StaticPagesSchemaMigration.php

```diff
@@ -38,15 +38,17 @@ public function up()
         Schema::create('static_page_settings', function (Blueprint $table) {
             $table->bigIncrements('static_page_setting_id');
             $table->bigInteger('static_page_id');
-            $table->foreign('static_page_id', 'static_page_settings_static_page_id')->references('static_page_id')->on('static_pages')->onDelete('cascade');
-            $table->index(['static_page_id'], 'static_page_settings_static_page_id');
+            $table->foreign('static_page_id', 'static_page_settings_static_page_id')
+                ->references('static_page_id')
+                ->on('static_pages')
+                ->onDelete('cascade');
+            $table->index(['static_page_id'], 'static_page_settings_static_page_id_index');
 
             $table->string('locale', 14)->default('');
             $table->string('setting_name', 255);
             $table->longText('setting_value')->nullable();
             $table->string('setting_type', 6)->comment('(bool|int|float|string|object)');
-            $table->index(['static_page_id'], 'static_page_settings_static_page_id');
-            $table->unique(['static_page_id', 'locale', 'setting_name'], 'static_page_settings_pkey');
+            $table->unique(['static_page_id', 'locale', 'setting_name'], 'static_page_settings_static_page_id_locale_setting_name_unique');
```

Comment:
Use the convention we've got elsewhere in the system for naming these -- it would be `static_page_settings_unique` in this case.

Reply:
fixed